sindresorhus/refined-github
# Avoid issue with conflict resolution

- Fixes #4673

## Test

0. Commit this:

```
Title
====================

Title
====================

Title
====================

Title
====================
```

1. Edit a line in a new branch
2. Edit the same line on `main`
3. Open PR from the branch, in my case https://github.com/fregante/sandbox/pull/29
4. See conflict


## Demo

In this demo I'm highlighting the lines I expect to be deleted. (Selection doesn't affect the behavior)

https://user-images.githubusercontent.com/1402241/130323477-e800493d-5617-4c8d-a2c9-239e639db9cc.mov



## Notes

GitHub’s own editor is confused:

<img width="278" alt="Screen Shot 3" src="https://user-images.githubusercontent.com/1402241/130322477-46c8c8a2-0f06-4519-a4c9-1cb10cb68779.png">


VSCode works correctly:

<img width="515" alt="Screen Shot 4" src="https://user-images.githubusercontent.com/1402241/130322489-c9334576-7818-4508-804e-3d7fb8844add.png">
